plugins/module_utils/net_map/networking_definition.py

```diff
@@ -894,6 +894,129 @@ def __str__(self):
         return f"{self.__start_ip}-{self.__end_ip}"
 
 
+class HostNetworkRoute(ansible_encoding.RawConvertibleObject):
```

Comment:
The `HostNetworkRange` inherits `RawConvertibleObject` because that way I can use the class as part of the exceptions we raise. If we are not going to pass instances of this class to an exception we are safe without this.
PS: I don't see where the abstract method is implemented.